CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+# How to contribute
```

Comment:
We could maybe upload this file to our new `docs` repo? 
It looks something generic enough to have it as template